.git-blame-ignore-revs

```diff
@@ -0,0 +1,5 @@
+# Reformat with Black
+53a7e3500437a9fd5826bc04758f7116bd7e52dc
+# ClangFormat the C code
+46b7e86bab79450ec0a2866c6c0c679afb659d17
+d374015504df2c7f0316154deade1bc1fa5075df
```

Comment:
Could you explain why it is necessary to include https://github.com/python-pillow/Pillow/commit/d374015504df2c7f0316154deade1bc1fa5075df? It is the merge commit for #4770. I would have expected https://github.com/python-pillow/Pillow/pull/4770/commits/5d968accf5db5d45fceeb68b9cbc4af44be0f6b4 instead, another commit from that PR.

Reply:
You're absolutely right, d374015 isn't required here, so got rid of it. I didn't add 5d968ac though, since it's a small non-mechanical commit.

Reply:
> I didn't add https://github.com/python-pillow/Pillow/commit/5d968accf5db5d45fceeb68b9cbc4af44be0f6b4 though, since it's a small non-mechanical commit.

👍